pytest_django/fixtures.py

```diff
@@ -323,3 +327,21 @@ def _live_server_helper(request):
     """
     if 'live_server' in request.funcargnames:
         request.getfuncargvalue('transactional_db')
+
+
+@pytest.fixture(scope='function')
+def assert_num_queries(db, pytestconfig):
```

Comment:
It looks like it should not use the `db` fixture, but some other that makes it work with `transactional_db`, too.
Speaking of this: please also add a test (maybe parameterized) for `transactional_db` (next to `db`).